package-lock.json

```diff
@@ -4841,22 +4841,25 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "bundled": true,
+          "resolved": false,
```

Comment:
Wondering what is different about your local environment that caused these changes to this file. Odd that it updated so much without an update to `package.json`. 

@csell5 any thoughts what might have caused these changes?

Reply:
no clue, environment shouldn't have changed from the weekend